src/components/organisms/ProjectPartners.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement } from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styles from './ProjectPartners.module.css'
+import Carousel from './Carousel'
+import Container from '../atoms/Container'
+import LinkOpener from '../molecules/LinkOpener'
+
+const query = graphql`
+  {
+    allFile(
+      filter: { absolutePath: { regex: "/src/images/projectPartners/" } }
+    ) {
+      edges {
+        node {
+          childImageSharp {
+            id
+            original {
+              src
+            }
+          }
```

Comment:
suggestion: for responsive image optimization we could / should probably make use of [gatsby-image](https://www.gatsbyjs.com/plugins/gatsby-image/#two-types-of-responsive-images)